src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -28,8 +28,18 @@
         line-height: 20px;
     }
 
+    &.bcs-is-scrolled::before {
+        position: absolute;
+        // position it behind bcs-content-header
+        top: -60px;
```

Comment:
What's the source of these values? Should they be variables shared between this component and `bcs-content-header`?